test/extended/default_forcePull_test.go

```diff
@@ -0,0 +1,262 @@
+// +build default
```

Comment:
don't use camelCase in filenames, the file name should be `default_force_pull_test.go`